services/httpd/handler_test.go

```diff
@@ -1078,7 +1080,9 @@ func TestHandler_Flux_QueryJSON(t *testing.T) {
 			t.Fatalf("unexpected query -got/+exp\n%s", cmp.Diff(c.Query, exp))
 		}
 		called = true
-		return internal.NewFluxQueryMock(), nil
+
+		p := &mock.Program{}
+		return p.Start(ctx, nil)
```

Comment:
You could  use `FluxQueryMock` if that implements `flux.Query`. Otherwise, I think you can just remove that.

Reply:
We added a mock package at some point to flux probably after the last time we updated this so I thought it was easier to just start using that since it would actually be maintained while the offshoot struct in the 1.x line of InfluxDB would likely get out of sync again.